packages/create-block/lib/templates/esnext/readme.txt.mustache

```diff
@@ -0,0 +1,54 @@
+=== {{title}} ===
+Contributors:      {{author}}
+Tags:              comments, spam
+Requires at least: 5.0
```

Comment:
I don't think it was tested with WordPress 5.0, 5.1 and 5.2 so we should either test it with older versions or set the minimum version to 5.3.